error-reporting/quickstart.js

```diff
@@ -14,16 +14,25 @@
 
 'use strict';
 
+// [START error_reporting_quickstart]
+// Import the Google Cloud client library
+const {ErrorReporting} = require('@google-cloud/error-reporting');
+
 function quickstart() {
-  // [START error_reporting_quickstart]
-  // Imports the Google Cloud client library
-  const {ErrorReporting} = require('@google-cloud/error-reporting');
+  try {
+    throw new Error('Something went wrong');
+  } catch (exception) {
+    reportError(exception);
+  }
+}
 
+function reportError(exception) {
   // Instantiates a client
   const errors = new ErrorReporting();
 
-  // Reports a simple error
-  errors.report('Something broke!');
-  // [END error_reporting_quickstart]
+  // Reports an exception
+  errors.report(exception.stack);
```

Comment:
**Question:**
In the equivalent [C# sample](https://github.com/GoogleCloudPlatform/dotnet-docs-samples/pull/2337/files#diff-5494443eba0482e62010d9445b529d4f511c98efeeb2c01bb512d71202798356L81) and the [Java sample](https://github.com/GoogleCloudPlatform/java-docs-samples/pull/8648/files#diff-f55c582b94d740b432da5ddc2be409846bea16d36c644b65796b8db2c12be6a4R71), we create a `ReportedErrorEvent` object, and report that to Google Cloud.
Is there a reason we don't use a `ReportedErrorEvent` object here (in our JavaScript)?

Reply:
This lib has handwritten method that abstracts the work developers have to do when using auto-generated libraries